### PR TITLE
Countdown timer 2

### PR DESCRIPTION
**Pull request by chiester**

### CODE REVIEW

**Comment by chiester** [comment]
Here's more detail about my proposed change to `parseTime()` in `date-utils`: 

Consider the case of the time `12:00PM`. This is noon. It's the 12th hour of the day in our 24-hour clock. However, `parseTime` applies the rule that if the string contains `PM`, then we add an offset of 12 hours. This works for hours 1-11, but clearly breaks in the case of `12:00PM`. If the rule were applied, `12:00PM` is declared to be the 24th hour of the day instead of the 12th hour of the day.

**Comment by ttypic** [comment]
@chiester, thank you very much! We will fix this soon

File: functions/utils/date-utils.ts [hunk 1]
```diff
@@ -60,6 +60,13 @@ const parseTime = (americanTime: string) => {
   const offset = americanTime.toUpperCase().endsWith('PM') ? PM_OFFSET : 0;
   const trimmedTime = americanTime.slice(0, americanTime.length - 2);
   const [hoursString, minutesString] = trimmedTime.split(':');
+
+  // handle exception cases of the midnight and noon hours
+  if (americanTime.startsWith('12')) {
```
Comment (ttypic) [inline functions/utils/date-utils.ts]: Fixed it and added test, I didn't realized that you fixed it too. Could you update branch once more?

**Review by chatfueldemeter — COMMENTED** [review]
For testing

**Pull request merged**